Multiplayer/Components/MainMenu/MultiplayerPane.cs

```diff
@@ -67,6 +69,7 @@ private void ShowPortPopup()
             return;
 
         popup.labelTMPro.text = Locale.SERVER_BROWSER__PORT;
+        popup.GetComponentInChildren<TMP_InputField>().text = Multiplayer.Settings.Port.ToString();
```

Comment:
This is the port for the server. The port should get saved per-server without a default option (saves when connecting).

Reply:
I could separate it into two settings one for "server port" and one for "default remote port" for now, if that would be preferred?

Reply:
Ok, these have now been separated into 2 categories, settings for hosting and details of the last server connected to by remote IP (only option until the server browser is operational)